src/lib/loaders/loaders_without_authentication/impulse.ts

```diff
@@ -0,0 +1,36 @@
+import factories from "../api"
```

Comment:
This file winds up serving as an example of how to allow data loaders to work via token swapping.